MAVProxy/modules/mavproxy_console.py

```diff
@@ -36,6 +36,8 @@ def __init__(self, mpstate):
         self.last_sys_status_errors_announce = 0
         self.user_added = {}
         self.safety_on = False
+        self.unload_check_interval = 0.1 # seconds
```

Comment:
on slow systems (eg. windows python) I suspect this will trigger while loading

Reply:
how about 0.5 seconds?

Reply:
and this parameter also used in mavproxy_map/mission_editor/param_editor